src/events/ready.ts

```diff
@@ -9,12 +9,5 @@ export default event({
         console.log(client.user.tag, 'is online!');
         // Send the message for role reactions
         sendReactionRoleMessage(client);
-        // Update the cache so that the bot can properly react to messages sent before it went live
-        updateCache(client)
```

Comment:
Does it need rebased (might be the wrong word 😅) to not remove this?

Reply:
Well the `updateCache` function is in #108 now, which is now merged, so I could just rebase the support feature branch to add it back

Reply:
Waaaait a minute, now I see what you mean. That shouldn't be there :thinking: What did I do? :joy:

Reply:
This is why you use rebase instead of merge, now I accidentally made a merge commit because of Github suggesting it and now it made changes I didn't want >.< Just gotta remove the merge commit

Reply:
Don't ever look at my commit history 😅 I think there's still a conflict, though

Reply:
Well that took a bit of doing but at least I got some much needed rebasing exercise 😅 I had to go back to the start of the branch and remove some edits there. I was able to rebase the branch successfully against main now, so there shouldn't be any conflicts